CONTRIBUTING.md

```diff
@@ -25,8 +25,16 @@ guidance below.
 ## Changing the code-base
 
 Generally speaking, you should fork this repository, make changes in your
-own fork, and then submit a pull-request. All new code should have associated unit
```

Comment:
Well, I suppose the Platform team (which is largely different than the V1 team) should make an affirmative decision about what it prefers.